src/quick-view.scss

```diff
@@ -29,24 +29,24 @@ $fd-quick-view-group-item-indent: 0.75rem;
     @include fd-reset();
 
     padding: 1rem;
-  }
 
-  .#{$fd-namespace}-bar {
-    padding: 0;
+    .#{$fd-namespace}-bar {
+      padding: 0;
 
-    &--header-with-subheader {
-      height: auto;
-      padding-bottom: $fd-quick-view-group-item-indent;
+      &--header-with-subheader {
+        height: auto;
+        padding-bottom: $fd-quick-view-group-item-indent;
 
-      .#{$fd-namespace}-avatar {
-        margin-right: $fd-quick-view-avatar-indent;
-      }
+        .#{$fd-namespace}-avatar {
+          margin-right: $fd-quick-view-avatar-indent;
+        }
 
-      .#{$fd-namespace}-bar {
-        &__left,
-        &__middle,
-        &__right {
-          align-items: flex-start;
```

Comment:
I don't see any change here except formatting

Reply:
@InnaAtanasova all `.#{$fd-namespace}-bar { ... }` block moved into `&__content {... <here now>}`

Reply:
ah, ok! Thanks for explaining. I totally missed this sneaky } :D